CONTRIBUTORS.md

```diff
@@ -0,0 +1,4 @@
+# ChiPy.org Contributors
+========================
+
+* [Bryan Chance](https://github.com/bhchance) - Added CONTRIBUTERS.md
```

Comment:
woo! first!

Reply:
Thanks! Can you add the following people as well:

[Adam 'Cezar' Jenkins](https://github.com/emperorcezar)
[Anjali Deolapure](https://github.com/adeolapure)
[Brian Ray](https://github.com/brianray)
[Cameron Rudnick](https://github.com/camrudnick)
[Carl Karsten](https://github.com/CarlFK)
[Dave Bertrand](https://github.com/db654)
[Jerry Dumblauskas](https://github.com/jerry-dumblauskas)
[Joe Jasinski](https://github.com/JoeJasinski)
[Ken Wasetis](https://github.com/ctxlken)
[Liam Schumm](https://github.com/lschumm)
[Malcolm Newsome](https://github.com/mnewsome)
[Raymond Berg](https://github.com/raymondberg)
[Ross Heflin](https://github.com/ne0zen)
[Sam Lahti](https://github.com/selecsosi)
[Tanya Schlusser](https://github.com/tanyaschlusser)
[Tathagata](https://github.com/tathagata)

Reply:
Sorry for the delay, added!